src/core/auth/dto/login.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { IsEmail, IsString, MinLength } from 'class-validator';
+
+export class LoginDto {
```

Comment:
Es
```suggestion
export class LoginDTO {
```
??

O realmente esta bien con
```suggestion
export class LoginDto {
```
??

He visto una forma en unas partes y de otra forma en otras, necesitamos consistencia XD